common/Makefile

```diff
@@ -1 +1,3 @@
+LDFLAGS = $(shell pkg-config opencv --libs)
+CFLAGS = $(shell pkg-config opencv --cflags)
```

Comment:
Why are these in common? OpenCV is component-specific, so shouldn't these flags only be in the makefile of whichever component needs them?

Reply:
Don't we need OpenCV datatypes for the message structs passed between components?